dataset_testing/testing_datasets.md

```diff
@@ -0,0 +1,27 @@
+
+# Playing with the nextGEMS cycle4 data
+
+Load your favorite python version, then create a venv for the easy life
+
+```bash
+python -m venv create easylife
+. easylife/bin/activate
+pip install --upgrade intake==0.7.0 easygems aiohttp jinja2 intake-xarray zarr pip setuptools ipykernel
+python -m ipykernel install --name easylife
+python
+```
```

Comment:
I would argue that we only advertise the use of mamba **or** venv to keep things simple (strong preference for mamba)